lib/web/ui/app.go

```diff
@@ -66,6 +66,9 @@ type App struct {
 	SAMLAppPreset string `json:"samlAppPreset,omitempty"`
 	// RequireRequest indicates if a returned resource is only accessible after an access request
 	RequiresRequest bool `json:"requiresRequest,omitempty"`
+	// UseProxyPublicAddrAsFQDN will rebuild this app's fqdn based on the proxy public addr that the
+	// request originated from.
+	UseProxyPublicAddrAsFQDN bool `json:"alwaysUseProxyPublicAddr,omitempty"`
```

Comment:
idk if it was overlooked or intentional, that the var and json field name are different

Reply:
overlooked, thank you! (the web UI is still looking for that old json name so it "worked", but ill update)